src/content/basics/secured-access-to-clusters/index.md

```diff
@@ -0,0 +1,79 @@
++++
+title = "Secured access to clusters for users and Giant Swarm support"
+description = "This documentation explains security measures for users and Giant Swarm support accessing offered services"
+date = "2019-05-09"
+weight = 45
+type = "page"
+categories = ["basics"]
++++
+
+# Secured access to clusters for users and Giant Swarm support
+
+In order to provide best service possible besides you having access to your clusters, also Giant Swarm staff will be able to connect.
+Thus it should be explained how this access differs and what security measures are brought into place so that your data is save with us. 
```

Comment:
```suggestion

Thus it should be explained how this access differs and what security measures are brought into place so that your data is saved with us. 
```